SignalR.RabbitMQ/RabbitConnection.cs

```diff
@@ -0,0 +1,74 @@
+using System;
+using System.Collections.Generic;
+using System.Linq.Expressions;
+using System.Threading.Tasks;
+using RabbitMQ.Client;
+using RabbitMQ.Client.Events;
+using RabbitMQ.Client.Framing;
+
+namespace SignalR.RabbitMQ
+{
+    internal class RabbitConnection : RabbitConnectionBase
+    {
+		private IConnection _rabbitConnection;
```

Comment:
Yeah... I know it's using tabs, but I don't know why it's using them inconsistently.  It also changed all the line endings in all of the .csproj files, which I went back and fixed.  I can reformat with spaces if need be.